### ISSUE TITLE
'Build failed' error to deploy to Apple Store. Test Mode Fail

### ISSUE THREAD

**Issue by pstatile** [FlutterFlow/flutterflow-issues#3413]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

HI! I am getting a 'Build failed :| Step 10 script `Flutter build ipa and automatic versioning` exited with status code 1' since an hour ago. I tried to include the version by hand but no luck. I saw that in the past this already happened. 
Also, the Test Mode is broken as well.
Could you check it? Thanks!

### Expected Behavior

Deploy to the apple store without errors and use test mode without problems

### Steps to Reproduce

Deploy to the apple store or launch test mode

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4KiMmBx89PsctJ0LXube9VvCYRKUA6U40WlsFuUQobfJTxG7YIRfSkP1ddZNPiSnw3PFv9kTge0/THhobhGuQHAzesfr5A1pZUSgmVIHOWacWgO8yCTnNfP91VHkuV0LejsxJSWcl0WFID3V2yBdmQThPaQe/2OVcsRfKkMprOo1bhAi+6Xn0Jg1ZSfCvr

### Visual documentation

<img width="1174" alt="Captura de pantalla 2024-07-16 a la(s) 2 36 59 p  m" src="https://github.com/user-attachments/assets/04b418e1-8946-45ac-a09e-aa0ddefb4c9c">

<img width="1462" alt="Captura de pantalla 2024-07-16 a la(s) 2 42 24 p  m" src="https://github.com/user-attachments/assets/59a6b551-5b0d-404f-b303-8a9541a2eb43">

### Environment

```markdown
- FlutterFlow version: v4.1.72+
- Platform: Web
- Browser name and version: Chrome
- Operating system and version affected: Mac
```

**Comment by Alezanello** [comment]
Hello!

I'm sorry to hear you're experiencing this issue. It seems that the Bug Report Code you provided is either incomplete or contains an error. Could you please generate a new code and send it to us? This will help us address the problem as quickly as possible.

Thank you!

**Comment by pstatile** [comment]
HI! Its:

IT4KiMmBx89PsctJ0LXube9VvCYRKUA6U40WlsFuUQobfJTxG7l3WvfObhJUcN/gYH9EJFn8+XMX/Mrsi+HEG/A3Bwqqc4g7/6lpWhaVZzqtb6bSEq63b3BTN/FXChG9yrSnvhRDHMhfdXwA2jn7O+utf3XXGITyO2oZReaoXdeK2SrDX1iXc2URm05KZDPz

Thanks!
Pablo

**Comment by pstatile** [comment]
Hi guys! Any news on this? I am blocked without able to send a working
build it to apple store.
thanks!

**Comment by Alezanello** [comment]
Hello!

Sorry for the delayed response. This seems to be an error related to `firebase_remote_config`. With the tools I have, I can't see further issues within this project or determine why it's failing to compile.

I strongly recommend contacting support via the in-app chat or at support@flutterflow.io so they can provide a more detailed solution. Additionally, I tried to run the project locally, but due to the length of the meditation pages, the FF CLI throws an error when downloading the project and run it locally

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/92ee7c1a-f966-4b3e-a5e3-044e9555e7f7/61a03859-62a6-4d5f-9812-410fc44ba271?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS85MmVlN2MxYS1mOTY2LTRiM2UtYTVlMy0wNDRlOTU1NWU3ZjcvNjFhMDM4NTktNjJhNi00ZDVmLTk4MTItNDEwZmM0NGJhMjcxIiwiaWF0IjoxNzIxMzEyODQyLCJleHAiOjMzMjkxODcyODQyfQ.Oa4fkw9x9be1afdihVOQxMO_QymuUm9MR3XKwws8o2U " alt="image.png" width="1454" height="213" />

Best regards,<br>Azanello

**Comment by pstatile** [comment]
Hi, I’ll do that. Additionally, I saw that a remote config error made me have a remote config duplicated. Could that be it? I can erase them and load them again if that helps. I’ll also try to shorten the pages names. Thanks!PabloEl 18 jul 2024, a la(s) 11:27, Alezanello ***@***.***> escribió:﻿
Hello!
Sorry for the delayed response. This seems to be an error related to firebase_remote_config. With the tools I have, I can't see further issues within this project or determine why it's failing to compile.
I strongly recommend contacting support via the in-app chat or at ***@***.*** so they can provide a more detailed solution. Additionally, I tried to run the project locally, but due to the length of the meditation pages, the FF CLI throws an error when downloading the project and run it locally

Best regards,Azanello
